# HG changeset patch
# User jwe@segfault.lan
# Date 1201922406 18000
#      Fri Feb 01 22:20:06 2008 -0500
# Node ID 22e4b22f07dce887d6dba88763cd931e95cd8f47
# Parent  b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8
Added tag ss-2-9-7 for changeset b2e1be30c8e9

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -15,8 +15,9 @@ e2ed74b9bfa052ce499b7f419e8d204123c49af0
 996a08a3eb068ce8002f54170fae755f9142d9a4 ss-2-9-0
 5c2f5830179082f18658d5a9e68087938444581e ss-2-9-1
 57b5030f5737ef9a1ac588b32e6c828358fac801 before-64-bit-merge
 9302581b820d71e6e3ae722b68e5923aebd46157 ss-2-9-2
 06585668a971b21818cacedbab1e11a98c11c5a0 ss-2-9-3
 f16c05db625059770b94eec0af027bbe74d2ce62 ss-2-9-4
 cda6a105ae9a30f9cd8032b1f49c2cfecfc04f6c before-ov-branch
 acb4a1e0b311c47bc957a48bf6af7f77e9c80fab ss-2-9-6
+b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8 ss-2-9-7
